core/services/relay/evm/cap_encoder.go

```diff
@@ -46,8 +48,22 @@ func NewEVMEncoder(config *values.Map) (consensustypes.Encoder, error) {
 		return nil, err
 	}
 
+	var subabi map[string]string
+	err = config.Underlying[subabiConfigFieldName].UnwrapTo(subabi)
```

Comment:
Could be shortened to:
```
type config struct {
    subapiConfigFieldName map[string]string
}
```

Reply:
Didn't make this change. May no longer be relevant because `subabi` is optional. Check this again.